flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/LogicalCorrelateToJoinFromTemporalTableFunctionRule.scala

```diff
@@ -97,7 +98,9 @@ class LogicalCorrelateToJoinFromTemporalTableFunctionRule
           .getUnderlyingHistoryTable
         val rexBuilder = cluster.getRexBuilder
 
-        val relBuilder = FlinkRelBuilder.of(cluster, getRelOptSchema(leftNode))
+        val relBuilder = FlinkRelMdNonCumulativeCost.THREAD_PLANNER.get()
```

Comment:
use `ShortcutUtils.unwrapContext(cluster)` instead

Reply:
Thanks for your comments, I changed it, waiting for unit test pass